api/migrations/versions/16830a790f0f_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
+
+Revision ID: 16830a790f0f
+Revises: 380c6aa5a70d
+Create Date: 2024-02-01 08:21:31.111119
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '16830a790f0f'
+down_revision = '380c6aa5a70d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('tenant_account_joins', schema=None) as batch_op:
```

Comment:
What about moving `current_tenant_id` to `accounts`? 
Setting a flag(current) on `tenant_account_joins` record may result in multiple data being `true` at the same time, lacking atomicity. However, setting `accounts.current_tenant_id` can ensure that an account is only bound to one current tenant.